.github/workflows/erlang.yml

```diff
@@ -0,0 +1,38 @@
+name: Erlang CI
+
+on: [push, pull_request]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        otp: ['23.3']
```

Comment:
It's the matrix intentional in case we want to test multiple versions at some point?

Reply:
More like to "make it look like the other projects", actually.